src/main/java/com/icthh/xm/uaa/service/mail/MailService.java

```diff
@@ -44,6 +29,25 @@
 import org.springframework.stereotype.Service;
 import org.springframework.ui.freemarker.FreeMarkerTemplateUtils;
 
+import javax.annotation.Resource;
```

Comment:
just sort import without separate section for java imports

Reply:
Fixed imports